aopy/analysis/base.py

```diff
@@ -1604,7 +1604,7 @@ def calc_mt_tfcoh(data, ch, n, p, k, fs, step, fk=None, pad=2, ref=False, imagin
             Default is False.
         imaginary (bool, optional): if True, compute imaginary coherence.
         return_angle (bool, optional): if True, also return the phase difference between 
-            the two channels. Default False.
+            the two channels. Default False. Angles correspond to phase differences from ch1 to ch0.
```

Comment:
additional explanation either in this line or above in the 'ch' line would be helpful. 
Suggestions: 

> if ch = [ch1, ch0], Angles correspond to phase differences from ch1 to ch0. 
> In line 1587: Order of the channel indices impacts the sign of the angles returned in return_angle is True.